mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_instance_and_srs_marc_bib_create_job_profile.sql

```diff
@@ -0,0 +1,304 @@
+	"name": "Default - Create instance",
+	"action": "CREATE",
+	"deleted": false,
+	"metadata": {
+		"createdDate": "2021-01-14T14:00:00.000",
+		"updatedDate": "2021-01-14T15:00:00.462+0000",
+		"createdByUserId": "00000000-0000-0000-0000-000000000000",
+		"updatedByUserId": "00000000-0000-0000-0000-000000000000"
+	},
+	"userInfo": {
+		"lastName": "System",
+		"userName": "System",
+		"firstName": "System"
+	},
+	"description": "This action profile is used with FOLIO''s default job profile for creating Inventory Instances and SRS MARC Bibliographic records. It can be edited, duplicated, or deleted.",
+	"folioRecord": "INSTANCE",
+	"childProfiles": [],
+	"parentProfiles": []
+}') ON CONFLICT DO NOTHING;
+
+INSERT INTO ${myuniversity}_${mymodule}.mapping_profiles (id, jsonb) values
+    ('991c0300-44a6-47e3-8ea2-b01bb56a38cc',
+    '{
+	"id": "991c0300-44a6-47e3-8ea2-b01bb56a38cc",
+	"name": "Default - Create instance",
+	"deleted": false,
+	"metadata": {
+		"createdDate": "2021-01-14T14:00:00.000",
+		"updatedDate": "2021-01-14T15:00:00.462+0000",
+		"createdByUserId": "00000000-0000-0000-0000-000000000000",
+		"updatedByUserId": "00000000-0000-0000-0000-000000000000"
+	},
+	"userInfo": {
+		"lastName": "System",
+		"userName": "System",
+		"firstName": "System"
+	},
+	"description": "This field mapping profile is used with FOLIO''s default job profile for creating Inventory Instances and SRS MARC Bibliographic records. It can be edited, duplicated, deleted, or linked to additional action profiles.",
+	"childProfiles": [],
+	"mappingDetails": {
+		"name": "instance",
+		"recordType": "INSTANCE",
+		"mappingFields": [{
+				"name": "discoverySuppress",
+				"path": "instance.discoverySuppress",
+				"value": "",
+				"enabled": "true",
+				"subfields": []
+			}, {
+				"name": "staffSuppress",
+				"path": "instance.staffSuppress",
+				"value": "",
+				"enabled": "true",
+				"subfields": []
+			}, {
+				"name": "previouslyHeld",
+				"path": "instance.previouslyHeld",
+				"value": "",
+				"enabled": "true",
+				"subfields": []
+			}, {
+				"name": "hrid",
+				"path": "instance.hrid",
+				"value": "",
+				"enabled": "false",
+				"subfields": []
+			}, {
+				"name": "source",
+				"path": "instance.source",
+				"value": "",
+				"enabled": "false",
+				"subfields": []
+			}, {
+				"name": "catalogedDate",
+				"path": "instance.catalogedDate",
+				"value": "",
+				"enabled": "true",
+				"subfields": []
+			}, {
+				"name": "statusId",
+				"path": "instance.statusId",
+				"value": "",
+				"enabled": "true",
+				"subfields": [],
+				"acceptedValues": {
+					"26f5208e-110a-4394-be29-1569a8c84a65": "Uncataloged",
+					"2a340d34-6b70-443a-bb1b-1b8d1c65d862": "Other",
+					"52a2ff34-2a12-420d-8539-21aa8d3cf5d8": "Batch Loaded",
+					"9634a5ab-9228-4703-baf2-4d12ebc77d56": "Cataloged",
+					"daf2681c-25af-4202-a3fa-e58fdf806183": "Temporary",
+					"f5cc2ab6-bb92-4cab-b83f-5a3d09261a41": "Not yet assigned"
```

Comment:
If default inventory instance status types are adjusted in the UI
https://github.com/folio-org/mod-inventory-storage/tree/master/reference-data/instance-statuses
Are the accepted mapping values easily adjusted in UI as well for this default job profile?

Reply:
@carolegodfrey It seems like UI downloads all ids-names accepted values from inventory/settings, so UI has an opportunity to change them.